README.md

```diff
@@ -26,13 +26,9 @@ __Note: This project supports Python version 3.x.x.__
 
 To start the application from the project root, run:
 ```bash
-python copis/client.py
-```
-Or, using the start script in that folder, run:
-```bash
 python copisclient.py
 ```
-*Note: These 2 commands can be run from any folder in the project directory tree.*
```

Comment:
just curious why  you removed this line?

Reply:
Oh, I deleted it because there's just one command now - but I can put in a note saying you can run copisclient from anywhere

Reply:
actually leave the readme alone. on my branch, I have this:

### Usage

To start the application from the project root, run:
```bash
python copisclient.py
```
Or, as a module:
```bash
python -m copisclient
```
*Note: These 2 commands can be run from any folder in the project directory tree.*

Reply:
sure - i'll revert my changes

Reply:
done!